Dockerfile fails to build ("Test: case_expr_extend -> ERROR!")

## Steps to reproduce the issue
- Clone this repo
- Try to build the docker image

## Expected behavior
- Docker image builds successfully


## Actual behavior
```
[100%] Building share/sf2/arith_map.v
[100%] Building share/sf2/cells_map.v
[100%] Building share/sf2/cells_sim.v

  Build successful.

mkdir -p /opt/yosys/bin
cp yosys yosys-config yosys-abc yosys-filterlib yosys-smtbmc /opt/yosys/bin
strip -S /opt/yosys/bin/yosys
strip /opt/yosys/bin/yosys-abc
strip /opt/yosys/bin/yosys-filterlib
mkdir -p /opt/yosys/share/yosys
cp -r share/. /opt/yosys/share/yosys/.
cd tests/simple && bash run-test.sh ""
make[1]: Entering directory '/yosys/tests/simple'
+ gcc -Wall -o /yosys/tests/tools/cmp_tbdata /yosys/tests/tools/cmp_tbdata.c
Test: arrays02 -> ok
Test: asgn_binop -> ok
Test: case_expr_extend -> ERROR!
Note: Make sure that 'iverilog' is an up-to-date git checkout of Icarus Verilog.
make[1]: *** [../tools/autotest.mk:10: case_expr_extend.sv] Error 1
make[1]: Leaving directory '/yosys/tests/simple'
make: *** [Makefile:786: test] Error 2
The command '/bin/sh -c cd /yosys  && make  && make install  && make test' returned a non-zero code: 2
ERROR: Service 'yosys' failed to build : Build failed
```

--- a/tests/simple/case_expr_extend.sv
+++ b/tests/simple/case_expr_extend.sv
@@ -1,7 +1,7 @@
 module top(
     output logic [5:0] out
 );
-always_comb begin
+initial begin
     out = '0;
     case (1'b1 << 1)
         2'b10: out = '1;

--- a/tests/simple/case_expr_query.sv
+++ b/tests/simple/case_expr_query.sv
@@ -1,7 +1,7 @@
 module top(
     output logic [5:0] out
 );
-always_comb begin
+initial begin
     out = '0;
     case ($bits (out)) 6:
     case ($size (out)) 6:

